### PR TITLE
#2373 added the contact form to submission only.

### LINKED ISSUE #2373: Contact Form on Submission Only
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by ajrbyers**
- Makes the contact form available on the when the front end is disabled
- Closes #2373

### CODE REVIEW

**Review by mauromsl — APPROVED** [review]

File: src/templates/admin/journal/submissions.html [hunk 1]
```diff
@@ -27,92 +27,32 @@
...
+                        <h3>{% trans 'Licences' %}</h3>
+                        <p>{{ request.journal.name }} {% trans "allows the following licences for submission" %}:</p>
+                        {% include "elements/license_block.html" %}
+                    {% elif item.title == 'sections' %}
+                        {% include "elements/section_block.html" %}
+                    {% elif item.get_display_text %}
+                        <h3>{{ item.title|capfirst }}</h3>
+                        {{ item.get_display_text|safe }}
+                    {% endif %}
+                    <hr />
```
Comment (mauromsl) [inline src/templates/admin/journal/submissions.html:53]: This change seems unrelated to this issue, has it leaked from another branch?

**Review by mauromsl — COMMENTED** [review]
Just a question inline

File: src/templates/admin/journal/submissions.html (hunk 1 above)
Comment (ajrbyers) [reply src/templates/admin/journal/submissions.html:53]: Yes and no - it updates the submissionly only version of this page to work with the submission items so I included it.
